pyccel/codegen/compiling/library_config.py

```diff
@@ -499,26 +521,25 @@ def install_to(self, pyccel_dirpath, installed_libs, verbose, compiler):
             library.
         """
         existing_installation = self._check_for_cmake_package('GFTL', 'Fortran', target_name = self.target_name)
+
         if existing_installation:
             installed_libs['gFTL'] = existing_installation
             return existing_installation
-        dest_dir = Path(pyccel_dirpath) / self._dest_dir
-        if not dest_dir.exists():
-            if verbose:
-                print(f">> Creating a link to {self._src_dir} in {dest_dir}")
-            os.symlink(self._src_dir, dest_dir, target_is_directory=True)
-
-        new_obj = CompileObj("gFTL", folder = "gFTL", has_target_file = False,
-                          include = (dest_dir / 'GFTL-1.13/include/v2',))
-        installed_libs['gFTL'] = new_obj
 
-        self._discovery_method = 'CMake'
         sep = ';' if sys.platform == "win32" else ':'
         CMAKE_PREFIX_PATH = os.environ.get('CMAKE_PREFIX_PATH', '').split(sep)
-        os.environ['CMAKE_PREFIX_PATH'] = ':'.join(s for s in (*CMAKE_PREFIX_PATH, str(dest_dir))
-                                                   if s and Path(s).exists())
 
-        return new_obj
+        gftl_installation = importlib.resources.files('pyccel.extensions.gftl_install')
+        gftl_installation_gen = importlib.resources.as_file(gftl_installation)
+        with gftl_installation_gen as f:
```

Comment:
Same comment as above. I believe we should only create the context manager in the `with` block:

```suggestion
        with importlib.resources.as_file(gftl_installation) as f:
```